<!DOCTYPE html>

<html class="no-js" lang="en-US">

	<head>

		<meta charset="UTF-8">
		<meta http-equiv="X-UA-Compatible" content="IE=edge">
		<meta name="viewport" content="width=device-width, initial-scale=1.0">

				
			<link rel="shortcut icon" href="wp-content/uploads/2017/07/Logo3-600.png">
		
		
		
		
			<link rel="apple-touch-icon-precomposed" href="wp-content/uploads/2017/07/Logo3-600.png">
		
				

		<title>Posters &#8211; RECOMB 2018</title>
<link rel='dns-prefetch' href='//cdnjs.cloudflare.com' />
<link rel='dns-prefetch' href='//fonts.googleapis.com' />
<link rel='dns-prefetch' href='//s.w.org' />
<link rel="alternate" type="application/rss+xml" title="RECOMB 2018 &raquo; Feed" href="feed/" />
<link rel="alternate" type="application/rss+xml" title="RECOMB 2018 &raquo; Comments Feed" href="comments/feed/" />
		<script type="text/javascript">
			window._wpemojiSettings = {"baseUrl":"https:\/\/s.w.org\/images\/core\/emoji\/11.2.0\/72x72\/","ext":".png","svgUrl":"https:\/\/s.w.org\/images\/core\/emoji\/11.2.0\/svg\/","svgExt":".svg","source":{"concatemoji":"http:\/\/recomb2018.fr\/wp-includes\/js\/wp-emoji-release.min.js?ver=5.1.1"}};
			!function(a,b,c){function d(a,b){var c=String.fromCharCode;l.clearRect(0,0,k.width,k.height),l.fillText(c.apply(this,a),0,0);var d=k.toDataURL();l.clearRect(0,0,k.width,k.height),l.fillText(c.apply(this,b),0,0);var e=k.toDataURL();return d===e}function e(a){var b;if(!l||!l.fillText)return!1;switch(l.textBaseline="top",l.font="600 32px Arial",a){case"flag":return!(b=d([55356,56826,55356,56819],[55356,56826,8203,55356,56819]))&&(b=d([55356,57332,56128,56423,56128,56418,56128,56421,56128,56430,56128,56423,56128,56447],[55356,57332,8203,56128,56423,8203,56128,56418,8203,56128,56421,8203,56128,56430,8203,56128,56423,8203,56128,56447]),!b);case"emoji":return b=d([55358,56760,9792,65039],[55358,56760,8203,9792,65039]),!b}return!1}function f(a){var c=b.createElement("script");c.src=a,c.defer=c.type="text/javascript",b.getElementsByTagName("head")[0].appendChild(c)}var g,h,i,j,k=b.createElement("canvas"),l=k.getContext&&k.getContext("2d");for(j=Array("flag","emoji"),c.supports={everything:!0,everythingExceptFlag:!0},i=0;i<j.length;i++)c.supports[j[i]]=e(j[i]),c.supports.everything=c.supports.everything&&c.supports[j[i]],"flag"!==j[i]&&(c.supports.everythingExceptFlag=c.supports.everythingExceptFlag&&c.supports[j[i]]);c.supports.everythingExceptFlag=c.supports.everythingExceptFlag&&!c.supports.flag,c.DOMReady=!1,c.readyCallback=function(){c.DOMReady=!0},c.supports.everything||(h=function(){c.readyCallback()},b.addEventListener?(b.addEventListener("DOMContentLoaded",h,!1),a.addEventListener("load",h,!1)):(a.attachEvent("onload",h),b.attachEvent("onreadystatechange",function(){"complete"===b.readyState&&c.readyCallback()})),g=c.source||{},g.concatemoji?f(g.concatemoji):g.wpemoji&&g.twemoji&&(f(g.twemoji),f(g.wpemoji)))}(window,document,window._wpemojiSettings);
		</script>
		<style type="text/css">
img.wp-smiley,
img.emoji {
	display: inline !important;
	border: none !important;
	box-shadow: none !important;
	height: 1em !important;
	width: 1em !important;
	margin: 0 .07em !important;
	vertical-align: -0.1em !important;
	background: none !important;
	padding: 0 !important;
}
</style>
	<link rel='stylesheet' id='wp-block-library-css'  href='wp-includes/css/dist/block-library/style.min.css?ver=5.1.1' type='text/css' media='all' />
<link rel='stylesheet' id='contact-form-7-css'  href='wp-content/plugins/contact-form-7/includes/css/styles.css?ver=5.1.1' type='text/css' media='all' />
<link rel='stylesheet' id='mptt-style-css'  href='wp-content/plugins/mp-timetable.htmlmedia/css/style.css?ver=2.2.1' type='text/css' media='all' />
<link rel='stylesheet' id='svg-flags-css-css'  href='wp-content/plugins/svg-flags-lite/flag-icon-css/css/flag-icon.min.css?ver=5.1.1' type='text/css' media='all' />
<link rel='stylesheet' id='svg-flags-plugin-css-css'  href='wp-content/plugins/svg-flags-lite/css/svg-flags.css?ver=5.1.1' type='text/css' media='all' />
<link rel='stylesheet' id='retro-fonts-css'  href='http://fonts.googleapis.com/css?family=Roboto+Slab%3A400%2C700%7COswald%3A400%2C700&#038;subset&#038;ver=5.1.1' type='text/css' media='all' />
<link rel='stylesheet' id='retro-icons-css'  href='wp-content/themes/Retro-4/fonts/retro-icons.css?ver=5.1.1' type='text/css' media='all' />
<link rel='stylesheet' id='retro-style-css'  href='wp-content/themes/Retro-4/style.css?ver=5.1.1' type='text/css' media='all' />
<link rel='stylesheet' id='tablepress-default-css'  href='wp-content/tablepress-combined.min.css?ver=19' type='text/css' media='all' />
<script type='text/javascript' src='wp-includes/js/jquery/jquery.js?ver=1.12.4'></script>
<script type='text/javascript' src='wp-includes/js/jquery/jquery-migrate.min.js?ver=1.4.1'></script>
<script type='text/javascript' src='wp-content/themes/Retro-4/js/modernizr.js?ver=2.7.1'></script>
<script type='text/javascript' src='//cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.4.1/jquery.easing.min.js?ver=5.1.1'></script>
<link rel='https://api.w.org/' href='wp-json/' />
<link rel="EditURI" type="application/rsd+xml" title="RSD" href="xmlrpc.php?rsd" />
<link rel="wlwmanifest" type="application/wlwmanifest+xml" href="wp-includes/wlwmanifest.xml" /> 
<meta name="generator" content="WordPress 5.1.1" />
<link rel="canonical" href="posters.html" />
<link rel='shortlink' href='?p=1161' />
<link rel="alternate" type="application/json+oembed" href="wp-json/oembed/1.0/embed?url=http%3A%2F%2Frecomb2018.fr%2Fposters%2F" />
<link rel="alternate" type="text/xml+oembed" href="wp-json/oembed/1.0/embed?url=http%3A%2F%2Frecomb2018.fr%2Fposters%2F&#038;format=xml" />
<noscript><style type="text/css">.mptt-shortcode-wrapper .mptt-shortcode-table:first-of-type{display:table!important}.mptt-shortcode-wrapper .mptt-shortcode-table .mptt-event-container:hover{height:auto!important;min-height:100%!important}body.mprm_ie_browser .mptt-shortcode-wrapper .mptt-event-container{height:auto!important}@media (max-width:767px){.mptt-shortcode-wrapper .mptt-shortcode-table:first-of-type{display:none!important}}</style></noscript><style>body, input, textarea, .blog-list .post-title h3, .blog-list h4.post-meta, .portfolio-list ul li h3, .single h2.post-title, .page h2.post-title { font-family: "Roboto Slab", sans-serif; }</style><style>.hentry-content h1, .hentry-content h2, .hentry-content h3, .hentry-content h4, .hentry-content h5, .hentry-content h6, .section-subtitle, .blog-list h4.post-meta, .single h4.post-meta, .widget h3, .comments h3, .comments h4, .comment-author, .more-posts { font-family: "Oswald", serif; }</style><link rel="icon" href="wp-content/uploads/2017/07/cropped-Logo3-600-32x32.png" sizes="32x32" />
<link rel="icon" href="wp-content/uploads/2017/07/cropped-Logo3-600-192x192.png" sizes="192x192" />
<link rel="apple-touch-icon-precomposed" href="wp-content/uploads/2017/07/cropped-Logo3-600-180x180.png" />
<meta name="msapplication-TileImage" content="wp-content/uploads/2017/07/cropped-Logo3-600-270x270.png" />
		<style type="text/css" id="wp-custom-css">
			

.light-text nav ul#main-nav li a{
  color: #FFFFFF;
text-shadow: 3px 3px 3px 3px rgba(0,0,0,0.2);
}


.header-inner{
  background: none;
  background-color: #6d3165;
}

/* Pour virer le grain 
.section-inner{
     background: none;
}*/

.sticky-note {
	background: transparent url('wp-content/uploads/2017/12/notes.png') no-repeat;
	width: 280px;
	height: 280px;
	padding: 60px 30px 30px 30px;
	overflow-y: scroll;
	text-align: center;
	float: right;
	margin-top:-20px;
	background-size: 100% 100%;
}

.sticky-note div {
	transform: rotate(-2.5deg);
}

.sticky-note a {
	border-bottom: 1px dotted;
}

.rouge {
	color:#b81067;
}

.paligne {
	text-align:justify;
}

.ItemCommittee img {
	margin-right: .5rem;
}


.sponsors-list {
	display: flex;
	flex-wrap: wrap;
	justify-content: center;
	align-items: center;
	margin: 0 auto;
	padding: 1rem;
}
.sponsors-list a {
	vertical-align: middle;
	padding: 1rem 1rem;
	max-width: 220px;
}


.sponsors-list img {
	width: 100%
}
		</style>
		    
    <link href="https://fonts.googleapis.com/css?family=PT+Sans:400,700" rel="stylesheet"> 

		<style>

@font-face {
  font-family: "BazarMedium";
  src: url("wp-content/themes/Retro-4/fonts/bazarmedium.eot");
  src: url("wp-content/themes/Retro-4/fonts/bazarmedium.eot?#iefix") format("embedded-opentype"),
       url("wp-content/themes/Retro-4/fonts/bazarmedium.woff") format("woff"),
       url("wp-content/themes/Retro-4/fonts/bazarmedium.ttf") format("truetype"),
       url("wp-content/themes/Retro-4/fonts/bazarmedium.svg#BazarMedium") format("svg");
  font-weight: normal;
  font-style: normal;
}

/* Fix ugly font rendering */
@media screen and (-webkit-min-device-pixel-ratio:0) {
    @font-face {
        font-family: "BazarMedium";
            src: url("wp-content/themes/Retro-4/fonts/bazarmedium.svg#BazarMedium") format("svg");
    }
}

h1, h2, h3, h4, h5, h6,
nav ul#main-nav li {
  font-family: "BazarMedium";
  text-transform: uppercase;
}




	#logo a {

		background-image: url(wp-content/uploads/2017/09/logo-4.png);

	}





	nav ul#main-nav li a {
		font-size: 100%;
	}



@import url('http://fonts.googleapis.com/css?family=PT+Sans:400,700');

.sticky-note{
  overflow-y: auto;
}

h1, h2, h3, h4, h5, h6, nav ul#main-nav li {
    font-family: 'PT Sans', sans-serif;
    font-weight: bold;
    text-transform: uppercase;
}

nav ul#main-nav li {
    font-weight: normal;
    /*text-transform: none;*/
}

.wpcf7-form input{
 max-width: 480px;
}

.wpcf7-form textarea{
 max-width: 480px;
}

.wpcf7-form{
 max-width: 500px;
 margin: 0 auto;
}


hr.top-dashed,
hr.bottom-dashed{
  display: none;
}

body{
  font-size:18px;
  background-color:#ece7f8;
}

h2 {
    font-size: 40px;
}

h2.section-title{
    font-weight: bold;
    font-size: 40px;
}

nav ul#main-nav li {
  font-size: 22px;
}

.retro-borders {
    text-shadow: 2px 3px #FCFAF7;
}

#satellites, #satellites th, #satellites td{
 border: 0; 
}
</style>
	</head>

	<body id="start" class="page-template-default page page-id-1161 fixed-header left-logo">
<header role="banner">

    <div class="header-inner light-text" style="background-color: #2d214b">

        <div class="container">

            <div class="row clear">

                
<div id="logo">

	<a href="index.html" title="RECOMB 2018" >

		<h1>RECOMB 2018</h1>

	</a>
	
</div>
                <nav role="navigation">

                    <ul id="main-nav" class="clear"><li id="menu-item-163" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="program.html">Program</a></li>
<li id="menu-item-896" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="keynotes.html">Keynotes</a></li>
<li id="menu-item-1178" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="committees.html">Committees</a></li>
<li id="menu-item-313" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="satellites.html">Satellites</a></li>
<li id="menu-item-2046" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="participants-and-awards.html">Participants and awards</a></li>
<li id="menu-item-354" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="key-dates.html">Key Dates</a></li>
<li id="menu-item-2073" class="menu-item menu-item-type-custom menu-item-object-custom"><a href="https://recomb-conference.github.io/recomb2019">RECOMB 2019</a></li>
</ul>
                </nav>

            </div><!-- row -->

        </div><!-- container -->

        <hr class="bottom-dashed">  

    </div><!-- header-inner -->   

</header>
			

	
	<main role="main">

		<section>

			<div class="section-inner section-post " style="background-color: #ECE7F8">
				
				<hr class="top-dashed"> 

				<div class="container">

					<div class="row clear">

						<article class="col col-12 tablet-full mobile-full">

							<h2 class="post-title">Posters</h2>

							<hr class="single-hr">

							<div class="hentry-content">
								
<table id="tablepress-6" class="tablepress tablepress-id-6">
<thead>
<tr class="row-1 odd">
	<th class="column-1">ID</th><th class="column-2">Title</th><th class="column-3">Authors</th>
</tr>
</thead>
<tbody class="row-hover">
<tr class="row-2 even">
	<td class="column-1">1</td><td class="column-2">ARKS: chromosome-scale human genome scaffolding with linked read kmers</td><td class="column-3">René Warren, Lauren Coombe, Jessica Zhang, Benjamin Vandervalk, Justin Chu, Shaun Jackman and Inanc Birol</td>
</tr>
<tr class="row-3 odd">
	<td class="column-1">2</td><td class="column-2">gNOME-Report: an interactive open source platform for prioritizing phenotype-associated and actionable genetic variants</td><td class="column-3">In-Hee Lee, Jose A. Negron and Sek Won Kong</td>
</tr>
<tr class="row-4 even">
	<td class="column-1">3</td><td class="column-2">Improving pairwise comparison of protein sequences with domain co-occurrence</td><td class="column-3">Christophe Menichelli, Olivier Gascuel and Laurent Bréhélin</td>
</tr>
<tr class="row-5 odd">
	<td class="column-1">4</td><td class="column-2">Extracting Information from Inaccurate Sequencing Data</td><td class="column-3">Gleb Goussarov, Ilse Cleenwerck, Mohamed Mysara, Natalie Leys, Peter Vandamme and Pieter Monsieurs</td>
</tr>
<tr class="row-6 even">
	<td class="column-1">5</td><td class="column-2">Fine-grained k-mer Feature Modification for Alignement-free NGS Sequence Compsrison</td><td class="column-3">Phanucheep Chotnithi and Atsuhiro Takasu</td>
</tr>
<tr class="row-7 odd">
	<td class="column-1">6</td><td class="column-2">pysster: Classification Of Biological Sequences By Learning Sequence And Structure Motifs With Convolutional Neural Networks</td><td class="column-3">Stefan Budach and Annalisa Marsico</td>
</tr>
<tr class="row-8 even">
	<td class="column-1">7</td><td class="column-2">Aindex: a perfect hash sequence index for the fast bioinformatics tools prototyping</td><td class="column-3">Aleksey Komissarov</td>
</tr>
<tr class="row-9 odd">
	<td class="column-1">8</td><td class="column-2">Chromosomer 2: reference-based scaffold arranging tool</td><td class="column-3">Gaik Tamazian and Aleksey Komissarov</td>
</tr>
<tr class="row-10 even">
	<td class="column-1">9</td><td class="column-2">Construction of optimal minimizers schemes</td><td class="column-3">Guillaume Marcais, Dan Deblasio and Carl Kingsford</td>
</tr>
<tr class="row-11 odd">
	<td class="column-1">10</td><td class="column-2">Multi-Index Bloom Filters: A probabilistic data structure for sensitive multi-reference sequence classification with multiple spaced seeds</td><td class="column-3">Justin Chu, Emre Erhan, Hamid Mohamadi, Ben Vandervalk, Jeffrey Tse, Sarah Yeo, Shaun Jackman, Ka Ming Nip, Rene Warren and Inanc Birol</td>
</tr>
<tr class="row-12 even">
	<td class="column-1">11</td><td class="column-2">Bioinformatics workflow system for massive sequencing data analysis</td><td class="column-3">Gunhwan Ko, Pan-Gyu Kim and Byungwook Lee</td>
</tr>
<tr class="row-13 odd">
	<td class="column-1">12</td><td class="column-2">Prediction of complete Hi-C contact maps from genomic sequence alone</td><td class="column-3">Christopher J.F. Cameron, Josée Dostie and Mathieu Blanchette</td>
</tr>
<tr class="row-14 even">
	<td class="column-1">13</td><td class="column-2">Construction of individual recombination maps using linked-read sequencing data</td><td class="column-3">Andreea Dreau, Felicity Jones, Vrinda Venu and Ludmila Gaspar</td>
</tr>
<tr class="row-15 odd">
	<td class="column-1">14</td><td class="column-2">Novel-X: novel sequence insertions detection using linked-reads</td><td class="column-3">Dmitrii Meleshko and Iman Hajirasouliha</td>
</tr>
<tr class="row-16 even">
	<td class="column-1">15</td><td class="column-2">Ripo_pipe: A ribosome profiling pipeline optimized for data analysis in metabolism</td><td class="column-3">Jordan Berg and Jared Rutter</td>
</tr>
<tr class="row-17 odd">
	<td class="column-1">16</td><td class="column-2">HapCHAT: Adaptive haplotype assembly for efficiently leveraging high coverage in long reads</td><td class="column-3">Stefano Beretta, Murray Patterson, Tobias Marschall, Marcel Martin, Simone Zaccaria, Gianluca Della Vedova and Paola Bonizzoni</td>
</tr>
<tr class="row-18 even">
	<td class="column-1">17</td><td class="column-2">Fast and accurate bisulfite alignment and methylation calling for mammalian genomes</td><td class="column-3">Jonas Fischer and Marcel Schulz</td>
</tr>
<tr class="row-19 odd">
	<td class="column-1">18</td><td class="column-2">Refscope: A reference sequence, prior-knowledge system used to assess 10X Genomic Chromium System efficiency in neutralizing repeat noise</td><td class="column-3">Claire Jubin, Aurélie Leduc, Florian Sandron, Céline Baulard, Aurélie Perin, Marc Delépine, Anne Boland, Vincent Meyer, Robert Olaso and Jean-François Deleuze</td>
</tr>
<tr class="row-20 even">
	<td class="column-1">19</td><td class="column-2">A framework for cost-constrained genome rearrangement under Double Cut and Join</td><td class="column-3">Pijus Simonaitis and Krister Swenson</td>
</tr>
<tr class="row-21 odd">
	<td class="column-1">20</td><td class="column-2">Minerva: An Alignment and Reference Free Approach to Deconvolve Linked-Reads for Metagenomics</td><td class="column-3">David Danko, Dmitrrii Meleshko, Daniela Bezdan, Chris Mason and Iman Hajirasouliha</td>
</tr>
<tr class="row-22 even">
	<td class="column-1">21</td><td class="column-2">Tigmint: Correct Assembly Errors Using Linked Reads From Large Molecules</td><td class="column-3">Shaun Jackman, Lauren Coombe, Justin Chu, Rene Warren, Ben Vandervalk, Sarah Yeo, Zhuyi Xue, Hamid Mohamadi, Joerg Bohlmann, Steven Jones and Inanc Birol</td>
</tr>
<tr class="row-23 odd">
	<td class="column-1">22</td><td class="column-2">Nanopore Sequencing Technology and Tools for Genome Assembly: Computational Analysis of the Current State, Bottlenecks, and Future Directions</td><td class="column-3">Damla Senol Cali, Jeremie Kim, Saugata Ghose, Can Alkan and Onur Mutlu</td>
</tr>
<tr class="row-24 even">
	<td class="column-1">23</td><td class="column-2">Crow: Fast and accurate expression estimation from long reads</td><td class="column-3">Laraib Malik, Avi Srivastava and Rob Patro</td>
</tr>
<tr class="row-25 odd">
	<td class="column-1">24</td><td class="column-2">Haplotype Assembly in Single-Cell DNA Sequencing Data</td><td class="column-3">Gryte Satas and Benjamin Raphael</td>
</tr>
<tr class="row-26 even">
	<td class="column-1">25</td><td class="column-2">OMICtools: Facilitate bioinformatics analyses by combining artificial and collective intelligence</td><td class="column-3">Emeric Dynomant, Clément Levin, Emeline Duquenne, Ulrich Moutoussamy, Elhadji Thioune, Cyrille Petit, Sébastien Faye, Nicolas Levacher, Fabien Pichon, Marion Denorme and Arnaud Desfeux</td>
</tr>
<tr class="row-27 odd">
	<td class="column-1">26</td><td class="column-2">Investigating the methylation associated with negative gene regulation through RNA-seq and RRBS across 20 tissues</td><td class="column-3">Won-Jun Lim, Kyoung Hyoun Kim and Namshin Kim</td>
</tr>
<tr class="row-28 even">
	<td class="column-1">27</td><td class="column-2">Identification of Transcription Factor Binding Sites using ATAC-seq</td><td class="column-3">Zhijian Li, Marcel Schulz, Martin Zenke and Ivan Costa</td>
</tr>
<tr class="row-29 odd">
	<td class="column-1">28</td><td class="column-2">Coupled RNA 5-methylcytosine and alternative splicing as revealed by quantitative analysis of RNA-BisSeq data at alternative isoform level</td><td class="column-3">Junfeng Liu and Zhihua Zhang</td>
</tr>
<tr class="row-30 even">
	<td class="column-1">29</td><td class="column-2">The genome wide study of estrogen receptor (ERa)-regulated enhancers characterization in breast cancer transcriptional program and the repressive role of SUMOlated glucocorticoid receptor (GR) via MegaTrans disassembly by integrating NGS data analysis</td><td class="column-3">Qi Ma</td>
</tr>
<tr class="row-31 odd">
	<td class="column-1">30</td><td class="column-2">Alkaloid Biosynthesis-Related Transcriptome and Metabolome Profiling in Poppy Plants</td><td class="column-3">Jae Hyeon Oh, Dowan Kim, Min-Seok Jung, Chang Kug Kim and Tae Ho Lee</td>
</tr>
<tr class="row-32 even">
	<td class="column-1">31</td><td class="column-2">ChIP-seq simulations reveal key sources of variations and suggest experimental design</td><td class="column-3">Vishaka Datta and Sridhar Hannenhalli</td>
</tr>
<tr class="row-33 odd">
	<td class="column-1">32</td><td class="column-2">Identification of transcriptional signatures for cell types from single-cell RNA-Seq</td><td class="column-3">Lynn Yi, Vasilis Ntranos, Pall Melsted and Lior Pachter</td>
</tr>
<tr class="row-34 even">
	<td class="column-1">33</td><td class="column-2">Identification of tissue-specific aging-related epigenetic marker using age prediction model</td><td class="column-3">Heeyeon Choi and Hojung Nam</td>
</tr>
<tr class="row-35 odd">
	<td class="column-1">34</td><td class="column-2">Construction of Gene Regulatory Networks with Secondary Structures via Multi-task Sparse Group Lasso</td><td class="column-3">Yiming Qin, Yaohua Hu, Pak Chung Sham, Zhou Zhou, Bin Yan, Junwen Wang and Jing Qin</td>
</tr>
<tr class="row-36 even">
	<td class="column-1">35</td><td class="column-2">Improved Normalization of Nanostring nCounter Gene Expression Data</td><td class="column-3">Ramyar Molania, Johann A Gagnon-Bartsch, Alexander Dobrovic and Terence P Speed</td>
</tr>
<tr class="row-37 odd">
	<td class="column-1">36</td><td class="column-2">ASGAL: Aligning RNA-Seq Data to a Splicing Graph to Detect Novel Alternative Splicing Events</td><td class="column-3">Luca Denti, Raffaella Rizzi, Stefano Beretta, Gianluca Della Vedova, Marco Previtali and Paola Bonizzoni</td>
</tr>
<tr class="row-38 even">
	<td class="column-1">37</td><td class="column-2">Regulation of splicing by the chromatin factor DNMT3B is associated with specific genomic features in mammary epithelial cells.</td><td class="column-3">Sebastien Lemaire, Louis Dulaurier, Hélène Polvèche and Didier Auboeuf</td>
</tr>
<tr class="row-39 odd">
	<td class="column-1">38</td><td class="column-2">CREAM: new feature selection approach for epigenomic profiles of cells</td><td class="column-3">Seyed Ali Madani Tonekaboni, Parisa Mazrooei, Benjamin Haibe-Kains and Mathieu Lupien</td>
</tr>
<tr class="row-40 even">
	<td class="column-1">39</td><td class="column-2">Stabilized Independent Component Analysis outperforms other methods in finding reproducible signals in tumoral transcriptomes</td><td class="column-3">Laura Cantini, Ulykbek Kairov, Aurélien de Reyniès, Emmanuel Barillot, Francois Radvanyi and Andrei Zinovyev</td>
</tr>
<tr class="row-41 odd">
	<td class="column-1">40</td><td class="column-2">Transfer of cell type information across single-cell datasets using adversarial neural networks</td><td class="column-3">Hui Ting Grace Yeo and David K Gifford</td>
</tr>
<tr class="row-42 even">
	<td class="column-1">41</td><td class="column-2">Probing instructions for expression regulation in gene nucleotide compositions</td><td class="column-3">Chloé Bessière, May Taha, Florent Petitprez, Jimmy Vandel, Jean-Michel Marin, Laurent Bréhélin, Sophie Lèbre and Charles-Henri Lecellier</td>
</tr>
<tr class="row-43 odd">
	<td class="column-1">42</td><td class="column-2">POSTIT: Multi-task learning to infer transcript isoform regulation from epi-genomics and transcriptomics data</td><td class="column-3">Azim Dehghani Amirabad and Marcel Schulz</td>
</tr>
<tr class="row-44 even">
	<td class="column-1">43</td><td class="column-2">Integrative analysis of single cell expression data reveals distinct regulatory states in bidirectional promoters</td><td class="column-3">Fatemeh Behjati Ardakani, Kathrin Kattler, Karl Nordstroem, Nina Gasparoni, Gilles Gasparoni, Sarah Fuchs, Anupam Sinha, Matthias Barann, Peter Ebert, Jonas Fischer, Barbara Hutter, Gideon Zipprich, Baerbel Felder, Juergen Eils, Benedikt Brors, Thomas Lengauer, Thomas Manke, Philip Rosenstiel, Joern Walter and Marcel Schulz</td>
</tr>
<tr class="row-45 odd">
	<td class="column-1">44</td><td class="column-2">Partially blind domain adaptation for age prediction from DNA methylation data</td><td class="column-3">Lisa Handl, Adrin Jalali, Michael Scherer and Nico Pfeifer</td>
</tr>
<tr class="row-46 even">
	<td class="column-1">45</td><td class="column-2">Inferring Gene Regulatory Programs with Mixtures of Sparse Multi-Task Regression Models</td><td class="column-3">Tobias Heinen, Azim Dehghani Amirabad, Florian Schmidt and Marcel Schulz</td>
</tr>
<tr class="row-47 odd">
	<td class="column-1">46</td><td class="column-2">Premature termination codon containing transcripts comprise a significant fraction of sequenced mRNAs</td><td class="column-3">Anna Hakobyan, Maria Nikoghosyan and Arsen Arakelyan</td>
</tr>
<tr class="row-48 even">
	<td class="column-1">47</td><td class="column-2">In-silico read normalization using set multi-cover optimization</td><td class="column-3">Dilip A. Durai and Marcel H. Schulz</td>
</tr>
<tr class="row-49 odd">
	<td class="column-1">48</td><td class="column-2">Cheatsheet for comparative epigenomic studies</td><td class="column-3">Claudia Chica, Alexandra Louis, Hugues Roest Crollius, Vincent Colot and François Roudier</td>
</tr>
<tr class="row-50 even">
	<td class="column-1">49</td><td class="column-2">Transcriptome reconstruction with quality score distortion in reference-based alignment</td><td class="column-3">Ana Hernandez</td>
</tr>
<tr class="row-51 odd">
	<td class="column-1">50</td><td class="column-2">GRASS-C - Graph-based RNA-Seq Analysis in Single cell level Subgraph Clustering</td><td class="column-3">Harry Taegyun Yang</td>
</tr>
<tr class="row-52 even">
	<td class="column-1">51</td><td class="column-2">Profiling adaptive immune repertoires across multiple human tissues by RNA Sequencing</td><td class="column-3">Serghei Mangul, Igor Mandric, Alex Zelikovsky and Eleazar Eskin</td>
</tr>
<tr class="row-53 odd">
	<td class="column-1">52</td><td class="column-2">GeFaST: An improved method for OTU assignment by generalising Swarm's fastidious clustering approach</td><td class="column-3">Robert Müller and Markus Nebel</td>
</tr>
<tr class="row-54 even">
	<td class="column-1">53</td><td class="column-2">Search for remote homologous sequences in large metagenomics datasets</td><td class="column-3">Romain Lannes, Philippe Lopez and Eric Bapteste</td>
</tr>
<tr class="row-55 odd">
	<td class="column-1">54</td><td class="column-2">Estimating the Total Genome Length of a Metagenomic Sample Using K-mers</td><td class="column-3">Kui Hua and Xuegong Zhang</td>
</tr>
<tr class="row-56 even">
	<td class="column-1">55</td><td class="column-2">Reference-guided genome assembly in metagenomic samples</td><td class="column-3">Cervin Guyomar, Wesley Delage, Fabrice Legeai, Mougel Christophe, Jean-Christophe Simon and Claire Lemaitre</td>
</tr>
<tr class="row-57 odd">
	<td class="column-1">56</td><td class="column-2">On the feasibility of mining CD8+ T cell receptor patterns underlying immunogenic peptide recognition</td><td class="column-3">Nicolas De Neuter, Wout Bittremieux, Charlie Beirnaert, Bart Cuypers, Aida Mrzic, Pieter Moris, Arvid Suls, Viggo Van Tendeloo, Benson Ogunjimi, Kris Laukens and Pieter Meysman</td>
</tr>
<tr class="row-58 even">
	<td class="column-1">57</td><td class="column-2">Systematic identification of cancer-specific immunogenic peptides with RAVEN</td><td class="column-3">Julia Sophia Gerke, Michaela Baldauf, Andreas Kirschner, Franziska Blaeschke, Manuel Effenberger, Kilian Schober, Rebeca Alba Rubio, Takayuki Kanaseki, Merve Kiran, Marlene Dallmayer, Julian Musa, Nusret Akpolat, Ayse Akatli, Fernando Rosman, Özlem Özen, Shinatro Sugita, Tadashi Hasegawa, Haruhiko Sugimura, Daniel Baumhoer, Maximilian Knott, Giuseppina Sannino, Aruna Marchetto, Jing Li, Dirk Busch, Tobias Feuchtinger, Shunya Ohmura, Martin Orth, Uwe Thiel, Thomas Kirchner and Thomas Grünewald</td>
</tr>
<tr class="row-59 odd">
	<td class="column-1">58</td><td class="column-2">Unsupervised computational deconvolution of mixed signals in tumor microenvironment using blind source deconvolution approach</td><td class="column-3">Urszula Czerwinska, Laura Cantini, Ulykbek Kairov, Emmanuel Barillot, Vassili Soumelis and Andrei Zinovyev</td>
</tr>
<tr class="row-60 even">
	<td class="column-1">59</td><td class="column-2">Modeling Drug Combinatorial Effects in Inflammatory Bowel Disease</td><td class="column-3">Lichy Han, Mateusz Maciejewski, Daniel Ziemek and Russ Altman</td>
</tr>
<tr class="row-61 odd">
	<td class="column-1">60</td><td class="column-2">An alignment-free method for the classification of diverse and complex virus groups</td><td class="column-3">Dylan Lebatteux, Mohamed Amine Remita and Abdoulaye Baniré Diallo</td>
</tr>
<tr class="row-62 even">
	<td class="column-1">61</td><td class="column-2">Model-based multiple variants test considering causal status</td><td class="column-3">Jong Wha Joo, Farhad Hormozdiari and Eleazar Eskin</td>
</tr>
<tr class="row-63 odd">
	<td class="column-1">62</td><td class="column-2">Evaluation of machine learning methods for genomic prediction of binary and continuous trait in plants</td><td class="column-3">Seongmun Jeong, Jae-Yoon Kim, Kyoung Hyoun Kim and Namshin Kim</td>
</tr>
<tr class="row-64 even">
	<td class="column-1">63</td><td class="column-2">Trajectory disease pattern mining incorporating with Exome-seq analysis for preterm biomarker detection</td><td class="column-3">Tun-Wen Pai</td>
</tr>
<tr class="row-65 odd">
	<td class="column-1">64</td><td class="column-2">1,000x Faster than PLINK: Genome-Wide Epistasis Detection with Logistic Regression Using Combined FPGA and GPU Accelerators</td><td class="column-3">Lars Wienbrandt, Jan Christian Kässens, Matthias Hübenthal and David Ellinghaus</td>
</tr>
<tr class="row-66 even">
	<td class="column-1">65</td><td class="column-2">PLEIOTROPY BETWEEN GWAS CATALOG TRAITS AND ALCOHOL-RELATED LIFE EVENTS AND SUBSTANCE-INDUCED DEPRESSIONS IN TWO INDEPENDENT POPULATIONS</td><td class="column-3">Qian Peng, Kirk Wilhelmsen and Cindy Ehlers</td>
</tr>
<tr class="row-67 odd">
	<td class="column-1">66</td><td class="column-2">Exploration of natural variability of gene expression</td><td class="column-3">Ani Sakhlyan, Anna Hakobyan and Arsen Arakelyan</td>
</tr>
<tr class="row-68 even">
	<td class="column-1">67</td><td class="column-2">JASS: visualising the results of a joint analysis in the context of GWAS</td><td class="column-3">Pierre Lechat, Hanna Julienne, Vincent Laville, Vincent Guillemot, Herve Menager and Hugues Aschard</td>
</tr>
<tr class="row-69 odd">
	<td class="column-1">68</td><td class="column-2">MentaLiST - MLST caller for large schemes with novel allele detection</td><td class="column-3">Pedro Feijao, Cedric Chauve, William Hsiao and Leonid Chindelevitch</td>
</tr>
<tr class="row-70 even">
	<td class="column-1">69</td><td class="column-2">Genome wide prediction of monoallelic gene expression from human epigenetic data</td><td class="column-3">Sivarajan Karunanithi and Marcel Schulz</td>
</tr>
<tr class="row-71 odd">
	<td class="column-1">70</td><td class="column-2">Novel strategy to map GWAS functional variants using sequencing in high-risk cases identifies putative risk-variants in myeloma</td><td class="column-3">Rosalie Waller, Karen Curtin, Djordje Antanackovic, Douglas Sborov, Steven Lipkin and Nicola Camp</td>
</tr>
<tr class="row-72 even">
	<td class="column-1">71</td><td class="column-2">Allelic imbalances and expression variability in GTEx Consortium data and gut organoids</td><td class="column-3">Svetlana Vinogradova, Saumya Gupta, Sébastien Vigneau and Alexander Gimelbrant</td>
</tr>
<tr class="row-73 odd">
	<td class="column-1">72</td><td class="column-2">From mutations to function: a pipeline to assess variation in the non-coding space</td><td class="column-3">Parisa Mazrooei, Paul Guilhamon, Tahmid Mehdi, Swneke Bailey and Mathieu Lupien</td>
</tr>
<tr class="row-74 even">
	<td class="column-1">73</td><td class="column-2">Investigating cancer risk germline mutations using eQTL networks</td><td class="column-3">Maud Fagny, John Platig, Marieke Kuijjer and John Quackenbush</td>
</tr>
<tr class="row-75 odd">
	<td class="column-1">74</td><td class="column-2">Pan-genome annotation transfer</td><td class="column-3">Jakub Tyrek, Paulina Dziadkiewicz and Norbert Dojer</td>
</tr>
<tr class="row-76 even">
	<td class="column-1">75</td><td class="column-2">Towards an Accurate and Efficient Heuristic for Species/Gene Tree Co-estimation</td><td class="column-3">Yaxuan Wang and Luay Nakhleh</td>
</tr>
<tr class="row-77 odd">
	<td class="column-1">76</td><td class="column-2">Powerful pathway enrichment and network analysis of GWAS summary data using GSA-SNP2</td><td class="column-3">Sora Yoon, Nguyen Hai and Dougu Nam</td>
</tr>
<tr class="row-78 even">
	<td class="column-1">77</td><td class="column-2">Integrated analysis of open accessible RNA expression data for Esophageal Squamous Cell Carcinoma</td><td class="column-3">Si Zheng, Hongyu Kang and Jiao Li</td>
</tr>
<tr class="row-79 odd">
	<td class="column-1">78</td><td class="column-2">Using Gene Expression to Predict Tumor Location in Prostate Cancer Tissue</td><td class="column-3">Osama Hamzeh, Abed Alkhateeb and Luis Rueda</td>
</tr>
<tr class="row-80 even">
	<td class="column-1">79</td><td class="column-2">Comprehensive Analysis of Tissue-wide Gene Expression and Phenotype Data Reveals Tissues Affected in Rare Genetic Disorders</td><td class="column-3">Ariel Feiglin, Bryce K. Allen, Isaac S. Kohane and Sek Won Kong</td>
</tr>
<tr class="row-81 odd">
	<td class="column-1">80</td><td class="column-2">Exploring genotype-phenotype associations to prioritize genetic variants implicated in digenic diseases</td><td class="column-3">Imane Boudellioua, Maxat Kulmanov, Paul N Schofield, Georgios V Gkoutos and Robert Hoehndorf</td>
</tr>
<tr class="row-82 even">
	<td class="column-1">81</td><td class="column-2">Drug repositioning for natural products using machine learning approaches</td><td class="column-3">Eunyoung Kim, A-Sol Choi and Hojung Nam</td>
</tr>
<tr class="row-83 odd">
	<td class="column-1">82</td><td class="column-2">Stepwise Elastic Net Regression with group penalties in anti-cancer drug response prediction with multiple genomic data</td><td class="column-3">Nan Li, Roman Schefzik, Bernd Fischer and Ângela Gonçalves</td>
</tr>
<tr class="row-84 even">
	<td class="column-1">83</td><td class="column-2">A systems biology approach to study interactions between interleukin 18 and different types of macrophages.</td><td class="column-3">Kaja Chmielewska, Dorota Formanowicz and Piotr Formanowicz</td>
</tr>
<tr class="row-85 odd">
	<td class="column-1">84</td><td class="column-2">Predication of chemotherapeutic response in bladder cancer</td><td class="column-3">Chia-Chen Tsai and Tzu-Hao Chang</td>
</tr>
<tr class="row-86 even">
	<td class="column-1">85</td><td class="column-2">Inference of Allele and Clone-specific Copy-Number Aberrations in Tumor Samples</td><td class="column-3">Simone Zaccaria and Benjamin Raphael</td>
</tr>
<tr class="row-87 odd">
	<td class="column-1">86</td><td class="column-2">Discovering Sensitizers to Overcome Cancer Drug Resistance</td><td class="column-3">Mijin Kwon and Doheon Lee</td>
</tr>
<tr class="row-88 even">
	<td class="column-1">87</td><td class="column-2">Modeling drug efficacy in cancer</td><td class="column-3">Krzysztof Koras, Anna Gambin, Dilafruz Juraeva, Johanna Mazur, Błażej Miasojedow, Eike Staub and Ewa Szczurek</td>
</tr>
<tr class="row-89 odd">
	<td class="column-1">88</td><td class="column-2">Cancer subtype discovery using multiple kernel learning based on clusters of features</td><td class="column-3">Nora K. Speicher and Nico Pfeifer</td>
</tr>
<tr class="row-90 even">
	<td class="column-1">89</td><td class="column-2">Predictive modeling and optimization of drug combinations</td><td class="column-3">Katarzyna Jabłonowska, Dilafruz Juraeva, Johanna Mazur, Eike Staub and Ewa Szczurek</td>
</tr>
<tr class="row-91 odd">
	<td class="column-1">90</td><td class="column-2">Integrative analysis of heterochromatin changes during ageing and carcinogenesis</td><td class="column-3">Vincent Piras, Mélanie Miranda, Marine Beinat, Mélody Matelot, Mickael Bourge, Romain Le Bars and Daan Noordermeer</td>
</tr>
<tr class="row-92 even">
	<td class="column-1">91</td><td class="column-2">Predicting Cancer Genes with Graph Convolutional Networks</td><td class="column-3">Roman Schulte-Sasse and Annalisa Marsico</td>
</tr>
<tr class="row-93 odd">
	<td class="column-1">92</td><td class="column-2">Panomics: Bayesian hierarchical context-dependent across-clustering for multi-omics pan-cancer data</td><td class="column-3">Magdalena Strauss, John Reid and Lorenz Wernisch</td>
</tr>
<tr class="row-94 even">
	<td class="column-1">93</td><td class="column-2">Early Detection of Preeclampsia Using Circulating Small non-coding RNA</td><td class="column-3">Liron Yoffe, Avital Gilam, Orly Yaron, Avital Polsky, Luba Farberov, Argyro Syngelaki, Kypros Nicolaides, Moshe Hod and Noam Shomron</td>
</tr>
<tr class="row-95 odd">
	<td class="column-1">94</td><td class="column-2">Tumor Phylogeny Reconstruction for Structural Variants</td><td class="column-3">Camir Ricketts, Daniel Seidman, Victoria Popic, Fereydoun Hormozdiari, Serafim Batzoglou and Iman Hajirasouliha</td>
</tr>
<tr class="row-96 even">
	<td class="column-1">95</td><td class="column-2">Loss-function learning for digital tissue deconvolution</td><td class="column-3">Franziska Görtler</td>
</tr>
<tr class="row-97 odd">
	<td class="column-1">96</td><td class="column-2">Efficient Algorithms to Discover Alterations with Complementary Functional Association in Cancer</td><td class="column-3">Rebecca Sarto Basso, Fabio Vandin and Dorit Hochbaum</td>
</tr>
<tr class="row-98 even">
	<td class="column-1">97</td><td class="column-2">Onctopus: Lineage-Based Subclonal Reconstruction</td><td class="column-3">Linda K. Sundermann, Daniel Doerr, Amit G. Deshwar, Jeff Wintersinger, Jens Stoye, Quaid Morris and Gunnar Rätsch</td>
</tr>
<tr class="row-99 odd">
	<td class="column-1">98</td><td class="column-2">Modeling the Effect of p16 on Two Types of Senescence</td><td class="column-3">Caroline Obermeier, Herbert Sizek and Erzsebet Regan</td>
</tr>
<tr class="row-100 even">
	<td class="column-1">99</td><td class="column-2">Subpoplar: reconstructing cancer phylogenies by ordering mutation pairs</td><td class="column-3">Jeff Wintersinger and Quaid Morris</td>
</tr>
<tr class="row-101 odd">
	<td class="column-1">100</td><td class="column-2">Methods for detecting contribution of mutational signatures in cancer genomes</td><td class="column-3">Damian Wójtowicz, Xiaoqing Huang, Yoo-Ah Kim and Teresa M. Przytycka</td>
</tr>
<tr class="row-102 even">
	<td class="column-1">101</td><td class="column-2">Inferring Parsimonious Migration Histories for Metastatic Cancers</td><td class="column-3">Mohammed El-Kebir, Gryte Satas and Ben Raphael</td>
</tr>
<tr class="row-103 odd">
	<td class="column-1">102</td><td class="column-2">Finding Consensus Phylogenetic Trees for Tumor Evolution</td><td class="column-3">Kiya Govek, Camden Sikes and Layla Oesper</td>
</tr>
<tr class="row-104 even">
	<td class="column-1">103</td><td class="column-2">Frequent Subgraph Mining of Dysregulated Pathways Predict Effective Cancer Therapeutics</td><td class="column-3">Gurkan Bebek and Arda Durmaz</td>
</tr>
<tr class="row-105 odd">
	<td class="column-1">104</td><td class="column-2">Integrated bioinformatics analyses of whole genome sequencing of major goat breeds by interpreting phylogenomics and selective sweep signals</td><td class="column-3">Jae-Yoon Kim, Seongmun Jeong, Kyoung Hyoun Kim and Namshin Kim</td>
</tr>
<tr class="row-106 even">
	<td class="column-1">105</td><td class="column-2">Fast Algorithms for the Cluster Containment Problem</td><td class="column-3">Andreas Dwi Maryanto Gunawan, Bingxin Lu and Louxin Zhang</td>
</tr>
<tr class="row-107 odd">
	<td class="column-1">106</td><td class="column-2">Keep your bacterial proteome comparisons Insyght</td><td class="column-3">Thomas Lacroix, Valentin Loux, Jonathan Lorenzo, Bryan Brancotte, Christophe Blanchet and Jean-François Gibrat</td>
</tr>
<tr class="row-108 even">
	<td class="column-1">107</td><td class="column-2">A Fast Adaptive Algorithm for Computing Whole-Genome Homology Maps</td><td class="column-3">Chirag Jain, Sergey Koren, Alexander Dilthey, Adam Phillippy and Srinivas Aluru</td>
</tr>
<tr class="row-109 odd">
	<td class="column-1">108</td><td class="column-2">Computational assessment of pathogenic non-coding genetic variants in the study of Mendelian diseases through an integrative supervised-learning approach</td><td class="column-3">Barthelemy Caron, Yufei Luo and Antonio Rausell</td>
</tr>
<tr class="row-110 even">
	<td class="column-1">109</td><td class="column-2">Ontology-based validation and identification of regulatory phenotypes</td><td class="column-3">Maxat Kulmanov, Paul N Schofield, Georgios V Gkoutos and Robert Hoehndorf</td>
</tr>
<tr class="row-111 odd">
	<td class="column-1">110</td><td class="column-2">mirLibSpark: a scalable NGS microRNA prediction pipeline with data aggregation</td><td class="column-3">Chao-Jung Wu, Mohamed Amine Remita and Abdoulaye Baniré Diallo</td>
</tr>
<tr class="row-112 even">
	<td class="column-1">111</td><td class="column-2">Structural Patterns in Plant MicroRNA Recognition</td><td class="column-3">Joanna Miskiewicz and Marta Szachniuk</td>
</tr>
<tr class="row-113 odd">
	<td class="column-1">112</td><td class="column-2">LCS-TA to assess similarity in RNA 3D structures</td><td class="column-3">Jakub Wiedemann, Tomasz Zok, Maciej Milostan and Marta Szachniuk</td>
</tr>
<tr class="row-114 even">
	<td class="column-1">113</td><td class="column-2">Using Deep Learning for prediction of protein abundance from RNA concentrations and Gene Ontology annotations</td><td class="column-3">Mitra Barzine, Alvis Brazma, Edgars Celms, Kārlis Čerāns, Jyoti Choudhary, Nuno Fonseca, Karlis Freivalds, Fatemeh Ghavidel, Andrew Jarnuczak, Lelde Lace, Darta Rituma and Juris Viksna</td>
</tr>
<tr class="row-115 odd">
	<td class="column-1">114</td><td class="column-2">A Deep Learning Approach for Predicting Protein-RNA Binding</td><td class="column-3">Ilan Ben-Bassat, Benny Chor and Yaron Orenstein</td>
</tr>
<tr class="row-116 even">
	<td class="column-1">115</td><td class="column-2">Detecting Circular RNA from High-throughput Sequence Data with De Bruijn Graph</td><td class="column-3">Xin Li and Yufeng Wu</td>
</tr>
<tr class="row-117 odd">
	<td class="column-1">116</td><td class="column-2">Augmented Geometry Representation enables Distinction of near native PPIs from decoys</td><td class="column-3">Mark Rozanov and Haim Wolfson</td>
</tr>
<tr class="row-118 even">
	<td class="column-1">117</td><td class="column-2">Inhibition of E-cadherin protein in human epithelial cells</td><td class="column-3">Hassan Hamed H Kaabi and Christopher Ward</td>
</tr>
<tr class="row-119 odd">
	<td class="column-1">118</td><td class="column-2">A full description of the conformational landscape of cyclized peptides using a robotics approach</td><td class="column-3">Maud Jusot, Jaysen Sawmynaden, Guillaume Postic, Marc Vaisset, Eric Ngo, Matthias Lerbinger, Théo Torcq, Dirk Stratmann, Jacques Chomilier and Juan Cortés</td>
</tr>
<tr class="row-120 even">
	<td class="column-1">119</td><td class="column-2">Design of cyclic peptide inhibitors of Caspase 3</td><td class="column-3">Jaysen Sawmynaden, Maud Jusot, Chahrazade El Amri, Damien Laage, Fabio Pietrucci, Dirk Stratmann and Jacques Chomilier</td>
</tr>
<tr class="row-121 odd">
	<td class="column-1">120</td><td class="column-2">Design of stable cyclic pentapeptides for the inhibition of caspase-2</td><td class="column-3">Guillaume Postic, Maxime Louet, Damien Laage, Fabio Pietrucci, Chahrazade El Amri, Jacques Chomilier and Dirk Stratmann</td>
</tr>
<tr class="row-122 even">
	<td class="column-1">121</td><td class="column-2">Improving the loop prediction and drug binding in GPCR structure models</td><td class="column-3">Bhumika Arora, K.V. Venkatesh, Denise Wootten and Patrick Sexton</td>
</tr>
<tr class="row-123 odd">
	<td class="column-1">122</td><td class="column-2">Pattern of codon usage for chemotaxis related protein genes in E.coli and T.maritima</td><td class="column-3">Serika Taga, Nobuyuki Uchikoga and Takanori Sasaki</td>
</tr>
<tr class="row-124 even">
	<td class="column-1">123</td><td class="column-2">Predicting anticancer peptides using machine learning algorithms</td><td class="column-3">Huei-Fang Lee and Chia-Yu Su</td>
</tr>
<tr class="row-125 odd">
	<td class="column-1">124</td><td class="column-2">Non-negative Matrix Factorization for Mass Spectrometry-Based Protein Identification</td><td class="column-3">Pasrawin Taechawattananant, Akiyasu C. Yoshizawa, Kazuyoshi Yoshii and Yasushi Ishihama</td>
</tr>
<tr class="row-126 even">
	<td class="column-1">125</td><td class="column-2">Identification of exon skipping events associated with Alzheimer’s disease in the human hippocampus</td><td class="column-3">Senggyun Han, Jason Miller, Seyoun Byun, Dokyoon Kim, Shannon Risacher, Andrew Saykin, Younghee Lee and Kwangsik Nho</td>
</tr>
<tr class="row-127 odd">
	<td class="column-1">126</td><td class="column-2">Identification of subgroups of patients with identical molecular characteristics by diffusing molecular profiles through a protein-protein interaction network</td><td class="column-3">Isabel Brito and Philippe Hupé</td>
</tr>
<tr class="row-128 even">
	<td class="column-1">127</td><td class="column-2">Integration Analysis for Immune System Inflection with Intermittent Fasting</td><td class="column-3">Soo Young Cho, Jong Kyu Woo, Soojun Park, Je Kyung Seong and Young Seek Lee</td>
</tr>
<tr class="row-129 odd">
	<td class="column-1">128</td><td class="column-2">A Novel Cell-Penetrating Peptide Modified Drug Targets Tumor Micorenvironment and Facilitates Therapeutic Internalization</td><td class="column-3">Margaret Dah-Tsyr Chang and Tun-Wen Pai</td>
</tr>
<tr class="row-130 even">
	<td class="column-1">129</td><td class="column-2">Incorporating prior knowledge into Mixed Graphical Model estimation</td><td class="column-3">Helena Zacharias, Karoline Lukaschek, Hamimatunnisa Johar, Michael Altenbuchinger, Gabi Kastenmüller, Kora Study Investigators, Karl-Heinz Ladwig and Jan Krumsiek</td>
</tr>
<tr class="row-131 odd">
	<td class="column-1">130</td><td class="column-2">Horizontal gene transfer can facilitate the formation of stable and diverse microbial communities: an in silico, agent-based model</td><td class="column-3">János Juhász, Attila Jády and Balázs Ligeti</td>
</tr>
<tr class="row-132 even">
	<td class="column-1">131</td><td class="column-2">Bagging procedure of the mutual rank index enhances effectiveness of gene coexpression information.</td><td class="column-3">Takeshi Obayashi, Yuichi Aoki and Kengo Kinoshita</td>
</tr>
<tr class="row-133 odd">
	<td class="column-1">132</td><td class="column-2">Structural and evolutionary analysis of barnyard grass chloroplast genome</td><td class="column-3">Ung-Han Yoon, Gang-Seob Lee, Tae-Ho Kim, Sung-Won Yoon and Keunpyo Lee</td>
</tr>
<tr class="row-134 even">
	<td class="column-1">133</td><td class="column-2">A hybrid combinatorial method for docking single stranded RNAs on proteins at the thermodynamic equilibrium</td><td class="column-3">Chinmay Singhal, Yann Ponty and Isaure Chauvot de Beauchene</td>
</tr>
<tr class="row-135 odd">
	<td class="column-1">134</td><td class="column-2">A non-procedural approach to predicting the 3D yeast genome from Hi-C data</td><td class="column-3">Kimberly MacKay, Mats Carlsson, Christopher Eskiw and Anthony Kusalik</td>
</tr>
<tr class="row-136 even">
	<td class="column-1">135</td><td class="column-2">Pan-genome structural analysis and visualisation</td><td class="column-3">Paulina Dziadkiewicz, Jakub Tyrek and Norbert Dojer</td>
</tr>
<tr class="row-137 odd">
	<td class="column-1">136</td><td class="column-2">Owning less is better than organizing more: investigation of multiple template modelling protocol of variable domains of Camelid antibodies</td><td class="column-3">Akhila Melarkode Vattekatte, Frédéric Cadet and Alexandre G . de Brevern</td>
</tr>
<tr class="row-138 even">
	<td class="column-1">137</td><td class="column-2">Deciphering the effects of proximity ligation in chromatin contact maps to understand single molecule interactions data</td><td class="column-3">Minji Kim, Meizhen Zheng, Simon Zhongyuan Tian, Byoungkoo Lee, Daniel Capurso, Jeffrey Chuang and Yijun Ruan</td>
</tr>
<tr class="row-139 odd">
	<td class="column-1">138</td><td class="column-2">GRAFENE: Graphlet-based alignment-free network approach integrates 3D structural and sequence (residue order) data to improve protein structural comparison</td><td class="column-3">Fazle Faisal, Khalique Newaz, Julie Chaney, Jun Li, Scott Emrich, Patricia Clark and Tijana Milenkovic</td>
</tr>
<tr class="row-140 even">
	<td class="column-1">139</td><td class="column-2">Studying the correlation between arsenicals and bladder, kidney and prostate cancer : a network analysis approach.</td><td class="column-3">Andrea Polo, Giuseppina De Petro, Alfredo Budillon and Susan Costantini</td>
</tr>
<tr class="row-141 odd">
	<td class="column-1">140</td><td class="column-2">Item Response Theory applied to Protein-Protein Interaction Networks reveals new structure and biological information</td><td class="column-3">Adrià Alcalá, Irene García and Mercè Llabrés</td>
</tr>
<tr class="row-142 even">
	<td class="column-1">141</td><td class="column-2">PINAWeb: A Web based tool to perform Protein Interaction Networks Alignments</td><td class="column-3">Ricardo Alberich, Adrià Alcalá, Mercè Llabrés and Gabriel Riera</td>
</tr>
<tr class="row-143 odd">
	<td class="column-1">142</td><td class="column-2">Promoter and enhancer chromatin dynamics during pancreatic differentiation</td><td class="column-3">Henriette Miko, Scott A. Lacadie and Uwe Ohler</td>
</tr>
<tr class="row-144 even">
	<td class="column-1">143</td><td class="column-2">Constructing Gene Regulatory Networks with Epigenetic data using Message-Passing</td><td class="column-3">Abhijeet Sonawane and Kimberly Glass</td>
</tr>
<tr class="row-145 odd">
	<td class="column-1">144</td><td class="column-2">cRegulome: An R package for accessing microRNA/transcription factor-gene expression correlations in cancer</td><td class="column-3">Mahmoud Ahmed and Deok Ryong Kim</td>
</tr>
<tr class="row-146 even">
	<td class="column-1">145</td><td class="column-2">Feedback from Cell Cycle Regulators to Plurioptency Factors May Explain the Inefficiency of Yamanaka Reprogramming</td><td class="column-3">Campbell Elliott, Micah Auerbach and Erzsebet Regan</td>
</tr>
<tr class="row-147 odd">
	<td class="column-1">146</td><td class="column-2">A Boolean Model of Neuronal Death Upon Cell Cycle Re-Entry</td><td class="column-3">Emma Schroeter and Erzsébet Ravasz Regan</td>
</tr>
<tr class="row-148 even">
	<td class="column-1">147</td><td class="column-2">Custom Definition and Visual Exploration of Metabolic Networks</td><td class="column-3">T. Cameron Waller, Jared Rutter and Alexander Lex</td>
</tr>
<tr class="row-149 odd">
	<td class="column-1">148</td><td class="column-2">Soft Vertex Classification for Active Module Problem</td><td class="column-3">Nikita Alexeev, Javlon Isomurodov and Alexey Sergushichev</td>
</tr>
<tr class="row-150 even">
	<td class="column-1">149</td><td class="column-2">MIIC online: a web server to reconstruct causal or non-causal networks from non-perturbative data</td><td class="column-3">Nadir Sella, Louis Verny, Guido Uguzzoni, Vincent Cabeli and Hervé Isambert</td>
</tr>
</tbody>
</table>
<!-- #tablepress-6 from cache -->
							</div>

						</article>  

						
					</div><!-- row -->           

				</div><!-- container -->

				<hr class="bottom-dashed">        

			</div>
			
		</section> 

	</main>


	<script type='text/javascript'>
/* <![CDATA[ */
var wpcf7 = {"apiSettings":{"root":"http:\/\/recomb2018.fr\/wp-json\/contact-form-7\/v1","namespace":"contact-form-7\/v1"}};
/* ]]> */
</script>
<script type='text/javascript' src='wp-content/plugins/contact-form-7/includes/js/scripts.js?ver=5.1.1'></script>
<script type='text/javascript' src='wp-content/themes/Retro-4/js/plugins.js?ver=4.9.2'></script>
<script type='text/javascript'>
/* <![CDATA[ */
var retro_home_vars = {"speed":"500","pausetime":"4000","autosliding":"slider-auto-sliding","pausehover":"slider-pause-hover","url":"http:\/\/recomb2018.fr\/wp-admin\/admin-ajax.php","ref":"375dbd8db9","articlenr":"3","portfolionr":"8"};
/* ]]> */
</script>
<script type='text/javascript' src='wp-content/themes/Retro-4/js/retro.js?ver=4.9.2'></script>
<script type='text/javascript' src='wp-includes/js/wp-embed.min.js?ver=5.1.1'></script>
<script type='text/javascript' src='wp-content/plugins/tablepress/js/jquery.datatables.min.js?ver=1.9.2'></script>
<script type="text/javascript">
jQuery(document).ready(function($){
$('#tablepress-6').dataTable({"order":[],"orderClasses":false,"stripeClasses":["even","odd"],"pagingType":"simple","lengthMenu":[10,20,25,50,100],"pageLength":20});
});
</script>			<script type="text/javascript">
				jQuery.noConflict();
				(function( $ ) {
					$(function() {
						// More code using $ as alias to jQuery
						$("area[href*=\\#],a[href*=\\#]:not([href=\\#]):not([href^='\\#tab']):not([href^='\\#quicktab']):not([href^='\\#pane'])").click(function() {
							if (location.pathname.replace(/^\//,'') == this.pathname.replace(/^\//,'') && location.hostname == this.hostname) {
								var target = $(this.hash);
								target = target.length ? target : $('[name=' + this.hash.slice(1) +']');
								if (target.length) {
								$('html,body').animate({
								scrollTop: target.offset().top - 180  
								},900 ,'easeInOutQuart');
								return false;
								}
							}
						});
					});
				})(jQuery);	
			</script>				
				
	</body>
	
</html>